collector/netdev.go

```diff
@@ -18,6 +19,10 @@ const (
 	netDevSubsystem = "network"
 )
 
+var (
+	fieldSep = regexp.MustCompile(" +|: *")
```

Comment:
Perhaps `"[ :] *"` is a bit easier to parse (for humans) and faster to process (for computers)?